app/src/main/java/org/gnucash/android/model/Transaction.java

```diff
@@ -268,7 +268,11 @@ public Money getBalance(String accountUID){
     public Money getImbalance(){
         Money imbalance = Money.createZeroInstance(mCurrencyCode);
         for (Split split : mSplitList) {
-            if (!split.getQuantity().getCommodity().getCurrencyCode().equals(mCurrencyCode)) {
+            Money quantity = split.getQuantity();
+            if (quantity.isAmountZero()) {
```

Comment:
In which case would we have a valid template transaction with zero quantity and non-zero value? It doesn't seem possible from the Android application. How do you create such template transaction?